eos-updater/eos-updater.service.in

```diff
@@ -7,6 +7,7 @@ After=network.target
 ExecStart=@bindir@/eos-updater
 Type=dbus
 BusName=com.endlessm.Updater
+IOSchedulingClass=idle
```

Comment:
Probably want to make the same change to
 * `eos-update-server.service.in` (it serves files over HTTP to P2P updating clients) and
 * `eos-updater-flatpak-installer.service.in` (in case flatpak updates are done without the system helper) too.

Reply:
Ah! I checked the update server - it's already there. I don't think we should do the flatpak installer, it blocks the successful first boot after an upgrade that installs a flatpak. In that case we expect heavy non-interactive contention so we should just throw all of the IO into the scheduler and let all of the startup jobs duke it out.

Reply:
Very well, thanks for checking and clarifying the decisions for future reference. :+1:

Carry on!